R/annotation.R

```diff
@@ -252,16 +263,26 @@ annotate_driver_ssm = function(maf_df,lymphoma_type,driver_genes,
 #' # Basic usage
 #' sv_df = get_manta_sv()
 #' annotated_sv = annotate_sv(sv_df)
-annotate_sv = function(sv_data,partner_bed,with_chr_prefix=FALSE,collapse_redundant=FALSE,return_as="bedpe",genome_build="grch37"){
-  bedpe1 = sv_data %>% dplyr::select("CHROM_A","START_A","END_A","tumour_sample_id","SOMATIC_SCORE","STRAND_A")
-  bedpe2 = sv_data %>% dplyr::select("CHROM_B","START_B","END_B","tumour_sample_id","SOMATIC_SCORE","STRAND_B")
+#'
+annotate_sv = function(sv_data, 
```

Comment:
Brett stumbled on a weird instance where a sample had a MYC-BCL2 translocation, and this function output two lines for the same breakpoint. One line had "BCL2-NA" as the fusion and the other had "NA-MYC". Any idea why that might happen?

Reply:
The function is designed only to handle oncogene-superenhancer rearrangements and this is neither.